src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio.mdx

```diff
@@ -33,115 +33,188 @@ redirects:
-
-      ```
-      repositories {
-        mavenCentral()
-      }
+dependencies {
+  implementation 'com.newrelic.agent.android:android-agent:<a href="/docs/release-notes/mobile-release-notes/android-release-notes">AGENT_VERSION</a>'
+}
+```
+
+In both scripts, replace `<AGENT_VERSION>` with the most recent agent version. See the [agent release notes](/docs/release-notes/mobile-release-notes/android-release-notes) to stay up to date. You may choose to configure one or both depending on your monitoring needs. 
+
+    </Step>
+    <Step>
+        ### Set app permissions 
+
+  Set app permissions to make requests to `INTERNET` and `ACCESS_NETWORK_STATE`. Add this code snippet to your `AndroidManifest.xml` file:
+
+```
+  <uses-permission android:name="android.permission.INTERNET" />
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+```
+
+        
+    </Step>
+    <Step>
+        ### Start the Android agent on app start up
+
+    In your **Default Activity**, import the `NewRelic` class. Your **Default Activity** is defined in your **Manifest**. 
+
+```
+<uses-permission android:name="android.permission.INTERNET" />
+<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+```
+
+To  initialize mobile monitoring, add the following script in your `onCreate()` method:
+
+```
+import com.newrelic.agent.android.NewRelic;
+```
+
+    </Step>
+</Steps>
+
+You may prefer completing the configuration process for a new mobile app later. To do that, go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile**, then select **See instructions** next to your mobile app name.
+
+If you want to upgrade an existing Android installation, then go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > (select an app) > Settings > Installation**.
+
+## Set up Gradle DSL with Gradle Plugin Portal [#gradle-dsl]
+
+<Callout variant="important">
+
+By default, the plugins `{} DSL` resolves itself from the [public Gradle Plugin Portal](https://plugins.gradle.org/). The Android agent is not yet available as a community plugin, but you can provide the classpath to [MavenCentral](https://mvnrepository.com/artifact/com.newrelic.agent.android/agent-gradle-plugin">MavenCentral) and still use the Plugins DSL.
+</Callout>
 
-      apply plugin: 'android'
-      apply plugin: 'newrelic'
+<Steps>
+    <Step>
+        ### Configure your `settings.gradle(.kts)` file for the agent plugin
 
-      dependencies {
-        implementation 'com.newrelic.agent.android:android-agent:<a href="/docs/release-notes/mobile-release-notes/android-release-notes">AGENT_VERSION</a>'
+<Tabs>
+	<TabsBar>
+        <TabsBarItem id="kts">
+            Configure with `settings.gradle(.kts)`
+        </TabsBarItem>
```

Comment:
@cthomas-newrelic Based on older content, it looks like this step is an either/or: 

1. you either use the settings.gradle file OR add to the project level of a build file.
2. Whatever the case, you always add the snippet on the app level

How do customers know if they want to use the kts file or the build file? What are the benefits/limitations of one over the other?